cli/src/test/java/com/liferay/blade/cli/command/GradlePrintErrorTest.java

```diff
@@ -61,12 +65,32 @@ public void testGradleError() throws Exception {
 
 		BladeTest bladeTest = bladeTestBuilder.build();
 
+		PowerMock.mockStaticPartialNice(BladeCLI.class, "getCommandMapByClassLoader");
+
+		InstallExtensionCommand command = new InstallExtensionCommand();
+
+		command.setArgs(new InstallExtensionArgs());
+
+		Map<String, BaseCommand<? extends BaseArgs>> commandMap = new HashMap<>();
+
+		commandMap.put("extension install", command);
+
+		IExpectationSetters<Map<String, BaseCommand<? extends BaseArgs>>> expect = EasyMock.expect(
+			BladeCLI.getCommandMapByClassLoader(EasyMock.anyString(), EasyMock.isA(ClassLoader.class)));
+
+		expect.andReturn(commandMap).atLeastOnce();
+
 		PowerMock.expectNew(
 			GradleExec.class, EasyMock.isA(BladeTest.class)).andReturn(new GradleExecSpecial(bladeTest));
 
-		PowerMock.replay(GradleExec.class);
+		PowerMock.replay(BladeCLI.class, GradleExec.class);
 
-		bladeTest.run(args);
+		try {
+			bladeTest.run(args);
+		}
+		catch (Throwable th) {
+			th.printStackTrace();
+		}
 
```

Comment:
We should remove the try / catch block and switch back to ust
```
bladeTest.run(args);
```
So our test will fail if there is a problem, rather than just printing the stacktrace.
I think I added this in testing and forgot to remove it.